encryptor/queryDataEncryptor_test.go

```diff
@@ -72,6 +72,76 @@ func normalizeQueryWithDialect(dialect dialect.Dialect, query string) (string, e
 	return sqlparser.StringWithDialect(dialect, parsed), nil
 }
 
+type parserTestData struct {
+	Query             string
+	QueryData         []interface{}
+	Normalized        bool
+	ExpectedQueryData []interface{}
+	Changed           bool
+	ExpectedIDS       [][]byte
+	DataCoder         DBDataCoder
+	dialect           dialect.Dialect
+}
+
+func testParsing(t *testing.T, testData []parserTestData, encryptedValue, defaultClientID []byte, schemaStore *config.MapTableSchemaStore) {
+	encryptor := &testEncryptor{value: encryptedValue}
+	parser := sqlparser.New(sqlparser.ModeStrict)
+	mysqlParser, err := NewMysqlQueryEncryptor(schemaStore, parser, encryptor)
```

Comment:
maybe lets name it just parser? because it is not `mysql` specific. and it is not a parser, it is encryptor.

Reply:
Will name it `queryEncryptor`.

Seems like there is no difference whether `NewMysqlQueryEncryptor` or `NewPostgresqlQueryEncryptor` is used in these tests, so probably no need in one more arg to `testParser()` to choose which QueryEncryptor to use.